portfolio/src/main/webapp/resume.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <title>Alvin Hermans</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono&display=swap">
+  <script src="script.js"></script>
```

Comment:
can you move these scripts to the bottom of `<body>`

Reply:
deleted instead, they weren't used on that page anyway but I can add back later if needed